src/layout/nav/Header.tsx

```diff
@@ -24,17 +25,27 @@ export type NavHeaderProps = {
     // for handling the closeAll action is within the parent object, and will
     // have an effect on navList property.
     onCloseAll?: () => void;
-
     primaryShown: boolean;
-
     secondaryShown: boolean;
+    style?: any;
+    color?: string;
```

Comment:
What color user suppose to give here ? If you want user to restrict Header color then your previous enum approach was correct. Just replace hard-coded class name values of previous enum with classes from Classnames.tsx